nodes/nodes/nodes-managing-bare-metal-nodes.adoc

```diff
@@ -0,0 +1,20 @@
+:context: bare-metal-nodes
+[id="managing-bare-metal-nodes"]
+= Managing bare metal nodes
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+When you install {product-title} on a bare metal cluster, you can provision and manage bare metal nodes using `machine` and `machineset` custom resources for bare metal hosts that exist in the cluster.
```

Comment:
```suggestion
When you install {product-title} on a bare metal cluster, you can provision and manage bare metal nodes using `machine` and `machineset` custom resources (CRs) for bare metal hosts that exist in the cluster.
```

The abbreviation "CR" is used later on, so we probably want to define it the first time it appears.